src/sdk/models/policies.ts

```diff
@@ -86,14 +86,18 @@ export interface PolicyTrackingGroupRule extends IEntityIdentifier<string> {
 export interface PolicyActionDescriptor extends IEntityIdentifier<string> {
 	name?: string;
 	description?: string;
+	destructive?: boolean;
+	exclusive?: boolean;
+	requires_existing?: boolean;
+	removes_item?: boolean;
+	prompt?: any;
```

Comment:
Figured this would eventually change to a specific type but leaving as `any` for now since it's out of scope of the current card.

Reply:
Works for me